packages/core/src/actors.ts

```diff
@@ -219,30 +236,43 @@ export function fromObservable<T, TEvent extends EventObject>(
 ): Behavior<TEvent, T | undefined> {
   let subscription: Subscription | undefined;
   let observable: Subscribable<T> | undefined;
-  const nextEventType = Symbol('next');
-  const errorEventType = Symbol('error');
-  const completeEventType = Symbol('complete');
   let canceled = false;
+  const nextEventType = '$$xstate.next';
+  const errorEventType = '$$xstate.error';
+  const completeEventType = '$$xstate.complete';
```

Comment:
Other special event types don't use the `$$` prefix. So those looks kinda out of sync with the rest of the codebase

Reply:
Yeah, open to suggestions here. Should we restrict `xstate.*` to only be for XState?